composer.json

```diff
@@ -23,7 +23,7 @@
   },
   "require-dev": {
     "johnkary/phpunit-speedtrap": "^3.0",
-    "mikey179/vfsstream": "^1.6",
+    "mikey179/vfsstream": "^1.6.8",
```

Comment:
@acoulton `mikey179/vfsstream` isn't compatible with PHP 7.4 until 1.6.8 (the latest release). Have I specified the requirement correctly or will composer now only track the point branches until 1.7? or will it continue to track up to ^2.x ? I can't remember how it treats the least significant part of the spec

Reply:
That's correct, `^` is all the way up to the next semver breaking, `~` would restrict to the level you specify e.g. it'd be `~1.6.8` that stops before 1.7

Reply:
Ahh perfect, couldn't remember which way round they were